example/jss/covid/toy/TruncatedPoisson.bl

```diff
@@ -5,25 +5,32 @@ model TruncatedPoisson {
   random IntVar realization
   
   /** Mean parameter \(\lambda\). \(\lambda > 0\) */
-  param RealVar mean
-  
+ param RealVar mean
+ random List<Double> factor ?: {
+ 	return new ArrayList(9) => [
+ 		for (int i : 3..<11){
+ 			add(exp(i * log(mean)))
+ 		}
+ 	]
+ }
   laws {
     logf(realization, mean) {
       if (mean <= 0) return NEGATIVE_INFINITY
       if (realization < 3 || realization > 11) return NEGATIVE_INFINITY
-      return realization * log(mean)
+     return -log(factor.sum())+realization * log(mean)
```

Comment:
I don't know whether this is an acceptable form of normalizing it, but it seems that it's not possible to call factor.sum() here.

Reply:
The trick here is to compute the normalization with a for loop. You will find the identity P(X >= 3, X <= 11) = 1 - sum_{x=3}^11 P(X = x) useful.

Reply:
Also, more general comment, it would be awesome to commit the data files in tidy format as well so that the code can be ran in a self-contained fashion.